process/phases.md

```diff
@@ -0,0 +1,150 @@
+ 
+## 0. Feature Proposal [CG]
+
+Entry requirements:
+  * A CG member has an idea.
+ 
+During this phase a new repository forking the spec repo is created (CG members
+can request this). PRs + Issues iterate on the design of the feature. If human
+readable design repo style .md files are useful, these are added to describe
+the feature as it's being designed. A tracking issue is created in the design
+repo.
+ 
+If relevant to demonstrate the viability of a feature, prototype
+implementations of the feature are implemented in interested engines (possibly
+on a branch).
+The CG will attempt to reach broad consensus.
+ 
+## 1. Proposed Spec Text Available [CG/WG]
+
+Entry requirements:
+   * Full proposed english spec text available in a forked repo around which a
+     reasonably high level of consensus exists.
+   * *Updates to the formal notation, test suite,
+      and reference interpreter are NOT yet required*.
+ 
+During this phase, one or more implementations proceed on prototyping the
+feature to the point that a comprehensive set of tests can be added. These
+tests need not pass the reference interpreter at this point, but should pass on
+some implementation.
+ 
+## 2. Implementation Phase [CG/WG]
+
+Entry requirements:
+   * Test suite has been updated to cover the feature in its forked repo.
+   * The test suite should run against some implementation, though it need not be
+     the reference interpreter.
+   * *Formal notation in the spec need not be updated.*
+ 
+During this phase, the following proceeds in parallel:
+Web VMs (browsers) implement the feature.
+The forked repo is updated to include revisions to the formalisation.
+The forked repo spec is updated to include implementation of the feature in the
+reference interpreter (OCaml).
+The feature is implemented in toolchains.
+ 
+## 3. Standardize the Feature [WG]
+
+Entry requirements:
+   * Two or more Web VMs implement the feature.
+   * At least one toolchain implements the feature.
+   * The formalisation and the OCaml reference interpreter are usually updated
+     (though these two can be done as part of step 3 at the WG chair's discretion).
+     CG + general consensus has continued to this point (as throughout).
+ 
+At this point the feature is fully handed off to the WG.
+During this phase, WG members discuss the feature, consider edge cases, and
+work towards consensus that the feature is now complete (kicking it back to an
+earlier stage in the CG if issues are uncovered).
+ 
+## 4. The Feature is Standardized [WG]
+
+Entry requirements:
+   * Consensus is reached in amongst WG members that the feature is complete.
+ 
+When Working Group consensus is reached (online), editors can merge the feature
+into master on the spec repo.
+The W3C snapshots (for REC) are made at a regular cadence (in a W3C repo), used
+to stamp official version. Matching tags are added in the github spec repo.
+The feature is enabled on stable by various browsers, converging on all.
```

Comment:
Domenic and others brought this one up, and my understanding is that these tags + versions carry special legal weight in terms of patent exclusion periods etc. (But I am not a lawyer).